utils.py

```diff
@@ -62,10 +68,25 @@ def assert_attachments(expected_count):
             print("{} attachments expected, {} found".format(expected_count,
                                                              attachment_count))
             sys.exit(1)
+    
+
+    def assert_group_membership(USER_ID, GROUP_ID):
```

Comment:
Can you change `USER_ID` to `user_id`? All caps is usually reserved to for global constants.